src/Internal/EventStorePersistentSubscription.php

```diff
@@ -104,7 +104,8 @@ public function startSubscription(
         ?UserCredentials $userCredentials,
         Closure $onEventAppeared,
         ?Closure $onSubscriptionDropped,
-        ConnectionSettings $settings
+        ConnectionSettings $settings,
+        Promise $stopped
```

Comment:
Where is this used? Also, why can't we use the `onSubscriptionDropped` callback?

Reply:
Answered in https://github.com/prooph/event-store-client/pull/117#issuecomment-619768573